sysml.library/Kernel Library/Objects.kerml

```diff
@@ -132,17 +135,5 @@ package Objects {
 		
 		derived feature redefines innerSpaceDimension = 
 			(if notEmpty(faces)? 2 else (if notEmpty(edges)? 1 else 0));
```

Comment:
Extra parentheses are not necessary. `if notEmpty(faces)? 2 else if notEmpty(edges)? 1 else 0` is equivalent and reads better.

Reply:
Thx, didn't know about the binding keyword. Pushed both changes.

Reply:
Pushed this change also.